posts/inside-rust/2020-03-19-all-hands-retrospective.md

```diff
@@ -0,0 +1,146 @@
+anyone that had a space to offer in a suitable time frame. We started looking
+at other options, like private hacker spaces and universities. The University
+of Macedonia in Thessaloniki, Greece made us a quite attractive offer, and
+they had the right time available, so we decided to go for that.
+
+### Budgeting and sponsorships.
+We in the All Hands group did our best to create a budget. Unfortunately we
+did not have access to any expense reports from the previous years. We had a
+survey we could use to estimate the number of attendees and a rough idea where
+they were coming from. We knew the costs of the venue. We added fudge factors.
+We thought we'd done a good job, though it would turn out that we were quite
+off in our calculations.
+
+Based on our estimated budget, it was clear that Mozilla was not going to be
+able to cover the costs alone. It seemed like a good opportunity to try and
+build relationships with some of the companies that have started to use Rust
+by approaching them for sponsorships. We weren't sure, though, how many
+sponsorships we'd be able to get, and so we hit upon the idea of using
+paid tickets.
+
+The idea was that people could purchase a ticket to the event, which would
+help to cover the costs, but that it was also okay to not have a ticket.
+Tickets have the advantage of being relatively easy for people to submit in
+their expense report if they are travelling with a company. We also found a
+number of companies who might be interested in a more traditional sponsorship.
+
+### Finding a bank account.
+Still, we had another problem. The Rust organisation does not have a bank
+account we could use to collect and distribute money, and we didn't have an
+obvious entity to sign legal agreements.
+
+At this point, we spoke to a company that might serve as a bank account. We
+had a quick call and went over a plan that seemed promising. Mozilla would
+contract with them for their services managing and organising the event, and
+this would serve as Mozilla's sponsorship. The other sponsorships and proceeds
+from ticket sales would collect in their bank account and we would
+manage distributions.
+
+### Pulling the trigger.
+At that point, we sent mail encouraging people to book tickets, and we also
+started following up more with sponsors. Our expectation was that we would
+soon have a bank account for collecting funds, and we started collecting
+information to repay people who had booked travel.
+
+In retrospect, it's clear that this was premature. It seemed like all the
+pieces were in place, and we knew that if people didn't start booking tickets,
+they never would. But the pieces were not in place, not actually, and we clearly
+should have taken this moment ask whether we were really going to be able to
+pull this off.
+
+### Realising it won't work.
+Finally, this all started to unravel in a about a month away from the event.
+In short, we encountered some logistical hurdles, which we won't go into, but
```

Comment:
I would expand a little - something like: business and legal issues which were insurmountable due to other organizations' constraints.